strongbox-security/strongbox-authentication-providers/strongbox-ldap-authentication-provider/src/main/java/org/carlspring/strongbox/authentication/api/ldap/CustomLdapUserDetailsMapper.java

```diff
@@ -0,0 +1,150 @@
+package org.carlspring.strongbox.authentication.api.ldap;
+
+import java.util.Base64;
+
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Perhaps it would be better to use Spring's implementation of the `StringUtils`. Are you using anything in particular that it doesn't support?